app/components/sections/hero-section.tsx

```diff
@@ -7,10 +7,12 @@ import {Button} from '../button'
 // Note that the image overlaps the right edge of the grid by `8vw`. This `8vw`
 // needs to stay in sync with the `10vw` margins of the grid component.
 function HeroSection() {
+  // The grid has a computed height on the large breakpoint, to make the Hero
+  // span the height of the screens, minus 10rem for the navbar (h-40).
   return (
-    <Grid className="h-full">
+    <Grid className="lg:h-[calc(100vh-10rem)] lg:pb-12">
```

Comment:
This is indeed a bracket thing while I just removed all of those in another commit. But I think it's the best way to do this. We need to subtract the navbar height from the screen height, to get the hero height.

Reply:
Yup, I have no problem with brackets for situations like this 👍